uint8 amount_segments
float32 angle_increment
float32[] ranges
int32[] start_points
int32[] stop_points
